api/src/main/java/jakarta/data/page/KeysetCursor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2022,2023,2024 Contributors to the Eclipse Foundation
```

Comment:
Copyrights need to follow the pattern of first-year-modified, latest-year-modified
```suggestion
 * Copyright (c) 2022,2024 Contributors to the Eclipse Foundation
```

Reply:
I ended up just dropping that commit.